### PR TITLE
Fix bug missing assignment of analysis module in ES-MDA

### PR DESCRIPTION
**Pull request by arielalmendral**

### CODE REVIEW

File: devel/python/python/ert_gui/models/connectors/run/multiple_data_assimilation.py [hunk 1]
```diff
@@ -50,8 +50,10 @@ def runSimulations(self):
         iteration_count = len(weights)
 
         print("Running MDA ES for %s  iterations\t%s" % (iteration_count, ", ".join(str(weight) for weight in weights)))
-        weights = self.normalizeWeights(weights)
 
+        self.setAnalysisModule()
```
Comment (joakim-hove) [inline devel/python/python/ert_gui/models/connectors/run/multiple_data_assimilation.py]: I do not understand this:

1. An analysis module has already been assigned - either the default module, or something from the configuration file.
2. I would assume the `setAnalysisModule()` method would require an argument?

**Comment by arielalmendral** [comment]
Set module does not need an argument, as currently implemented. It picks the "current choice" from the UI: 
```
def setAnalysisModule(self):
 module_name = AnalysisModuleModel().getCurrentChoice()
...
```
ES-MDA has opened the possibility of picking different modules now.

You may a see similar assignment for the other two schemes: ensemble_smoother and iterated_ensemble_smoother

**Comment by jepebe** [comment]
I think this functionality was addressed in the mega PR #1158

**Comment by arielalmendral** [comment]
OK, closing this one then.

**Pull request closed without merging**